reportportal/v5/templates/uat-deployment.yaml

```diff
@@ -37,6 +37,8 @@ spec:
           value: "minio"
         - name: RP_BINARYSTORE_MINIO_ENDPOINT
           value: "{{ .Values.minio.endpoint }}"
+        - name: RP_BINARYSTORE_MINIO_REGION
+          value: "{{ .Values.minio.region }}"
```

Comment:
Is it OK to bypass value `"null"` to the environment variable? It's not an empty value, nor an absent value. It's just a string `null`. I doubt that region `null` is a default region in minio.

Reply:
I just consulted with dev team, you'll break deployment. You should not set `null` value into environment variables. You must specify `RP_BINARYSTORE_MINIO_REGION` only if you have a valid value.

Reply:
Please check feedback from Andrey above
This was already noted